recipes/ocl/build.sh

```diff
@@ -0,0 +1,19 @@
+mkdir build -p
+cd build 
+
+if [[ ${HOST} =~ .*linux.* ]]; then
+	# https://github.com/conda-forge/boost-feedstock/issues/72
+	sed -i '127s#;##g' ${PREFIX}/include/boost/python/detail/caller.hpp
+fi
+
+cmake -G "Ninja" \
+      -D CMAKE_BUILD_TYPE:STRING=Release \
+      -D CMAKE_PREFIX_PATH:FILEPATH=$PREFIX \
+      -D CMAKE_INSTALL_PREFIX:FILEPATH=$PREFIX \
```

Comment:
Try putting the above in quotes, either `CMAKE_INSTALL_PREFIX:FILEPATH="${PREFIX}"` or `"CMAKE_INSTALL_PREFIX:FILEPATH=${PREFIX}"`. Also try removing `":FILEPATH"` from these.

Reply:
Hmm, as far as I can remember this was never a problem. But I will try.

Reply:
ok,  problem is related to cmake 3.15 (worked previously with cmake 3.14.)

cmake 3.15 breaks
```
find_package(Python3 ...)
```
for conda environments